src/vs/workbench/contrib/terminal/browser/xterm/quickFixAddon.ts

```diff
@@ -99,7 +129,27 @@ export class TerminalQuickFixAddon extends Disposable implements ITerminalAddon,
 		if (!terminal || !commandDetection) {
 			return;
 		}
-		this._register(commandDetection.onCommandFinished(command => this._resolveQuickFixes(command)));
+		this._register(commandDetection.onCommandFinished(command => {
+			if (this._fixesAvailable) {
+				if (this._fixesShown) {
+					if (this._expectedCommands?.includes(command.command)) {
+						this._logService.debug(QuickFixCommandResultTelemetryEvent.QuickFixExecutedManually);
+						this._telemetryService?.publicLog2<{ classification: 'SystemMetaData'; purpose: 'FeatureInsight' }>(QuickFixCommandResultTelemetryEvent.QuickFixExecutedManually);
+					} else {
+						this._logService.debug(QuickFixCommandResultTelemetryEvent.QuickFixWrong);
+						this._telemetryService?.publicLog2<{ classification: 'SystemMetaData'; purpose: 'FeatureInsight' }>(QuickFixCommandResultTelemetryEvent.QuickFixWrong);
```

Comment:
There's a little nuance here, opening the link is reported as wrong even if you open it manually. Should we just not send this for the PR link? This might just work itself out if we merge `_fixesAvailable` and `_expectedCommands`